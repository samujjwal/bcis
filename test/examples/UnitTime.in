unit second
unit minute = 60 * second
unit hour = 60 * minute

set x = 1 hour
print(x)