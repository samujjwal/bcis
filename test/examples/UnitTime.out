3600.0*second
